qriollo.rb

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
```

Comment:
Shouldn't need this line any more. Believe these days we enforce UTF-8 on all formula files at load.